ui/redux/reducers/settings.js

```diff
@@ -2,11 +2,16 @@ import * as ACTIONS from 'constants/action_types';
 import * as SETTINGS from 'constants/settings';
 import moment from 'moment';
 import SUPPORTED_LANGUAGES from 'constants/supported_languages';
+import { ACTIONS as LBRY_REDUX_ACTIONS, SHARED_PREFS } from 'lbry-redux';
 
 const reducers = {};
 const defaultState = {
   isNight: false,
   loadedLanguages: [...Object.keys(window.i18n_messages), 'en'] || ['en'],
+  customWalletServers: [],
+  sharedPrefs: {
+    [SHARED_PREFS.WALLET_SERVERS]: false,
```

Comment:
Left a comment on the redux PR but this can just use `DAEMON_SETTINGS`

Reply:
We talked about that - not sure if we should still change this. leaving it for now.